### PR TITLE
awsdac 0.21.2 (new formula)

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/a/awsdac.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  desc "CLI tool for drawing AWS architecture"
+  homepage "https://github.com/awslabs/diagram-as-code"
+  url "https://github.com/awslabs/diagram-as-code/archive/refs/tags/v0.21.2.tar.gz"
+  sha256 "cef4fe205423158de2db250a0356ebab8942747116d58bf487231d66ef3a4d84"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "cmd/awsdac/main.go"
```
Comment (chenrui333) [inline Formula/a/awsdac.rb:11]:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/awsdac"
```

File: Formula/a/awsdac.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+class Awsdac < Formula
+  desc "CLI tool for drawing AWS architecture"
+  homepage "https://github.com/awslabs/diagram-as-code"
+  url "https://github.com/awslabs/diagram-as-code/archive/refs/tags/v0.21.2.tar.gz"
+  sha256 "cef4fe205423158de2db250a0356ebab8942747116d58bf487231d66ef3a4d84"
+  license "Apache-2.0"
```
Comment (chenrui333) [inline Formula/a/awsdac.rb:6]:
```suggestion
  license "Apache-2.0"
  head "https://github.com/awslabs/diagram-as-code.git", branch: "main"
```

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@ugwis, thanks for your first contribution to Homebrew-core!! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Comment by ugwis** [comment]
Thank you, too!

**Pull request merged**